src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -131,6 +131,18 @@ def create_random_name(self, prefix, length):
 
         return moniker
 
+    def create_guid(self):
```

Comment:
This should fix the long standing issue that playback will not work if the command create a guid which can be part of url

Reply:
Can you add comments to the method to explain the use case for this method? The name of the method is too generic to indicate its usage.